assets/l10n/app_en.arb

```diff
@@ -529,5 +529,13 @@
   "manyPeopleTyping": "Several people are typing…",
   "@manyPeopleTyping": {
     "description": "Text to display when there are multiple users typing."
+  },
+  "messageIsEditedLabel": "EDITED",
+  "@messageIsEditedLabel": {
+    "description": "Label for an edited message."
```

Comment:
```suggestion
    "description": "Label for an edited message. (Use ALL CAPS for cased alphabets: Latin, Greek, Cyrillic, etc.)"
```

(similarly below)

Reply:
One thing I was wondering about when implementing this, is that whether we should programmatically upper case the letters and translate the label normally, or just do it the current way.

We do have a precedent: "loginMethodDivider": "OR". But I can also see the appeal to using `i18n.toUpperCase` for something purely for styling purposes (similar to the `text-transform: uppercase;` approach as one could do in web), because in this case we aren't using uppercase letters to convey emphasis/exaggeration in the writing.

Reply:
Started a discussion [here](https://chat.zulip.org/#narrow/stream/58-translation/topic/Requiring.20translation.20in.20uppercase.20for.20applicable.20languages/near/1931579). My guess is that it might be preferred to just use uppercase letters, but I'm curious if there are better ways.